HowTos/security_bulletin_faq.rst

```diff
@@ -216,6 +216,24 @@ How is data encrypted during transmission from source Controller to destination
 
 By default, data transfer is over a TCP connection with TLSv1.2 for encryption. Customers have the option to downgrade the TLS Version used due to internal dependency conflicts. You can configure this in Aviatrix Controller by clicking on Settings > Advanced > Security.
 
+How does Aviatrix encrypt data in transit? 
+--------------------------------------------------------------------------------------------
+Aviatrix 6.5 and above implements the spiffee framework which use a PKI/X.509 strong encryption. You can read more about this framework at spiffee.io. 
+
+How does Aviatrix handle security patch?
+--------------------------------------------------------------------------------------------
+Security patch resolves software vulnerabilities and will be patched to the compatible software versions as stated in the release notes. When a patch is released, there will be a field notice to Aviatrix Controller via email. In addition, customer can send an email to securityalerts@aviatrix.com with a subject Subscribe, to subscribe to our customer mailing list. 
```

Comment:
```suggestion
A security patch resolves software vulnerabilities and will be applied to the compatible software versions as stated in the release notes. When a patch is released, there will be a field notice to Aviatrix Controller via email.
```

Reply:
We should also remove the reference to SPIFFE